mainnet/14-wrong-permissions/evidence/against.md

```diff
@@ -0,0 +1,15 @@
+The proposal Justification says:
```

Comment:
Nit: should we prefix all the `wysiwyg` dispute dirs with that prefix? (similar to how we did for app mining)

Reply:
Second thought, this dir should probably be called `wrong-installation`

Reply:
Yep, let's do it so it's less confusing.

Reply:
Done in https://github.com/aragon/court-precedence-campaign/pull/3/commits/d187dc1b0d51aa49674a62a8f3a24a855da0d9a7